Database/Corrections/sodQuestFixes.lua

```diff
@@ -2928,6 +2928,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -402852,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90196] = {
+            [questKeys.name] = "Between the Eyes",
+            [questKeys.startedBy] = {{1543,1544},{405201}},
```

Comment:
The spawn of 405201 should be cleaned up to be a single one.

I think we should also simply remove the NPC spawns for this one.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5583/commits/e36697d7e3f475ffb04467eb424d38ee3b15f700